src/types/GameConstants.ts

```diff
@@ -31,7 +31,7 @@ export interface Queue {
   /**
    * Additional notes about the queue.
    */
-  notes: string | null;
+  notes: string | undefined;
```

Comment:
Same as above.

Reply:
So, can I replace nulls for undefined in here? https://github.com/TheDrone7/shieldbow/blob/22946eb1fb1193b7d23c8412fbffa348e4107766/src/client.ts#L155

Reply:
There aren't any, there.

Reply:
![image](https://user-images.githubusercontent.com/23432278/185771205-9d79a5f1-6511-4f45-89ae-ea7d237cb46c.png)

Reply:
It should actually be `string | null` but any `null` in the `types` directory is basically going to stay because of things like https://static.developer.riotgames.com/docs/lol/queues.json for which the structure is defined there.